tajo-plan/src/main/java/org/apache/tajo/plan/logical/AlterTableNode.java

```diff
@@ -42,6 +42,8 @@
   @Expose
   private KeyValueSet properties = new KeyValueSet();
   @Expose
+  private String[] propertyKeys;
```

Comment:
It would be better to change this variable to be more intuitive as well. I think getter and setter also need to be changed together.